modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/test/java/com/liferay/dynamic/data/mapping/internal/upgrade/v2_0_0/DDMFormInstanceUpgradeProcessTest.java

```diff
@@ -14,32 +14,39 @@
 
 package com.liferay.dynamic.data.mapping.internal.upgrade.v2_0_0;
 
+import com.liferay.dynamic.data.mapping.BaseDDMTestCase;
 import com.liferay.portal.kernel.model.ResourceAction;
 import com.liferay.portal.kernel.service.ResourceActionLocalService;
 import com.liferay.portal.kernel.util.ListUtil;
 import com.liferay.portal.model.impl.ResourceActionImpl;
+import com.liferay.portal.test.rule.LiferayUnitTestRule;
 
 import java.util.List;
 import java.util.function.Supplier;
 
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
 import org.junit.Test;
-import org.junit.runner.RunWith;
 
 import org.mockito.Matchers;
 
-import org.powermock.api.mockito.PowerMockito;
-import org.powermock.modules.junit4.PowerMockRunner;
-
 /**
  * @author Pedro Queiroz
  */
-@RunWith(PowerMockRunner.class)
-public class DDMFormInstanceUpgradeProcessTest extends PowerMockito {
+public class DDMFormInstanceUpgradeProcessTest extends BaseDDMTestCase {
+
+	@ClassRule
+	@Rule
+	public static final LiferayUnitTestRule liferayUnitTestRule =
```

Comment:
Is there any particular need for this rule to be declared?

Reply:
Also SF